### PR TITLE
Add singular translations for candidate and job-ad

### PR DESCRIPTION
**Pull request by 99999989**
### Before
Screenshot 2019-06-05 at 13 39 11

### After
Screenshot 2019-06-05 at 13 38 59

And the same for "xx Candidates" -> "1 Candidate"

### CODE REVIEW

File: alv-portal-ui/src/app/candidate-search/candidate-search/candidate-search.component.html [hunk 1]
```diff
@@ -33,7 +33,9 @@ <h2 class="sr-only">
       <div class="alv-bar mb-1">
         <span *ngIf="resultsAreLoading$ | async; else resultsCount">...</span>
         <ng-template #resultsCount>
-          <strong>{{totalCount$ | async | number }} {{'portal.candidate-search.results-count.label' | translate}}</strong>
+          <strong *ngIf="totalCount$ | async as totalCount">
```
Comment (birom) [inline alv-portal-ui/src/app/candidate-search/candidate-search/candidate-search.component.html]: Do we intentionally hide this, if no candidate has been found?

File: alv-portal-ui/src/app/job-advertisement/job-ad-search/job-search/job-search.component.html [hunk 2]
```diff
@@ -32,7 +32,10 @@ <h2 class="sr-only">
         <span *ngIf="resultsAreLoading$ | async; else resultsCount"
               data-test="count-placeholder">...</span>
         <ng-template #resultsCount>
-          <strong><span data-test="resultCount">{{totalCount$ | async | number}} </span>{{'portal.job-ad-search.results-count.label' | translate}}</strong>
+          <strong *ngIf="totalCount$ | async as totalCount">
```
Comment (birom) [inline alv-portal-ui/src/app/job-advertisement/job-ad-search/job-search/job-search.component.html]: We should not hide the text if the result list is empty.

File: alv-portal-ui/src/app/candidate-search/candidate-search/candidate-search.component.html (hunk 1 above)
Comment (99999989) [reply alv-portal-ui/src/app/candidate-search/candidate-search/candidate-search.component.html]: Fixed

File: alv-portal-ui/src/app/job-advertisement/job-ad-search/job-search/job-search.component.html (hunk 2 above)
Comment (99999989) [reply alv-portal-ui/src/app/job-advertisement/job-ad-search/job-search/job-search.component.html]: Fixed

File: alv-portal-ui/src/app/candidate-search/candidate-search/candidate-search.component.ts [hunk 3]
```diff
@@ -165,4 +165,9 @@ export class CandidateSearchComponent extends AbstractSubscriber implements OnIn
       this.cdRef.detectChanges();
     }
   }
+
+  getTotalCountLabel(totalCount: string): string {
+    return totalCount === '1' ? 'portal.candidate-search.results-count.label.singular' :
```
Comment (birom) [inline alv-portal-ui/src/app/candidate-search/candidate-search/candidate-search.component.ts]: Is 0 in German singular or plural? I think in Egnlish plural..

**Pull request merged**